appsec/tests/integration/src/test/bin/enable_extensions.sh

```diff
@@ -24,6 +24,7 @@ if [[ -f /appsec/ddappsec.so && -d /project ]]; then
     echo datadog.appsec.helper_path=/appsec/libddappsec-helper.so
     echo datadog.appsec.helper_log_file=/tmp/logs/helper.log
     echo datadog.appsec.helper_log_level=info
+#    echo datadog.appsec.helper_log_level=debug
```

Comment:
should this be not commented out?

Reply:
Unfortunately debug is too verbose because of the WAF. It's good for debugging, but not more normal tests runs I think.